mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_WebViewPopup.java

```diff
@@ -103,6 +104,11 @@ public boolean shouldOverrideUrlLoading(final WebView view, final String url) {
                         openDialogPopup();
                         return false;
                     }
+                    if (uriHost.equals("appleid.apple.com")) {
+                        Log.d("popup: AppleID login");
+                        openDialogPopup();
+                        return false;
+                    }
```

Comment:
![image](https://github.com/IITC-CE/ingress-intel-total-conversion/assets/9021449/b679ae04-8fb6-417e-a40c-ef9a827ae1b5)

Do i have to comment or is the image enough? 🙈

Reply:
You mean to combine these comditions into one? I think it's better to leave it as it is, because it makes the code more readable

Reply:
Well, this is horrible.
It can be done much better.
What about enclosing the conditions into methods and using one if like

```
If (isFacebook(url) || isGoogle(URL) || isNiantic(URL)) {
  Console.log("login URL encountered")
  Popup stuff....
}
```